index.js

```diff
@@ -66,7 +66,13 @@ module.exports = fp(function from (fastify, opts, next) {
     request({ method: req.method, url, qs, headers, body }, (err, res) => {
       if (err) {
         this.request.log.warn(err, 'response errored')
-        this.send(err)
+        if (!this.sent) {
+          if (err.code === 'ERR_HTTP2_STREAM_CANCEL') {
```

Comment:
Not sure whether this is correct way or not. I am trying to avoid exposing internal error message which contains host address information of target.
Please suggest.

Reply:
Error is already logged on line 68. Which should have all information.